projects/pnpm.io/package.yml

```diff
@@ -8,13 +8,15 @@ versions:
 
 provides:
   - bin/pnpm
+  - bin/pnpx
 
 dependencies:
-  nodejs.org: '^14,^16,^18'
+  nodejs.org: '>=14 <20'
```

Comment:
@jhheider gunna merge this, but be aware in case it causes issues! In which case pls fix.

Reply:
Won't find fixes if we don't break stuff. Let's do it.